Libraries/LibWeb/Painting/ImagePaintable.cpp

```diff
@@ -75,11 +75,20 @@ void ImagePaintable::paint(PaintContext& context, PaintPhase phase) const
             auto bitmap_aspect_ratio = (float)bitmap_rect.height() / bitmap_rect.width();
             auto image_aspect_ratio = (float)image_rect.height().value() / image_rect.width().value();
 
+            // FIXME: Scale may be wrong if device-pixels != css-pixels.
             auto scale_x = 0.0f;
             auto scale_y = 0.0f;
             Gfx::IntRect bitmap_intersect = bitmap_rect;
 
             auto object_fit = m_is_svg_image ? CSS::ObjectFit::Contain : computed_values().object_fit();
```

Comment:
Maybe add a link to https://drafts.csswg.org/css-images/#the-object-fit ?

Reply:
Added this link and another one for object-position further below.